add-on/chrome/modules/MozLoopService.jsm

```diff
@@ -1312,11 +1313,30 @@ this.MozLoopService = {
 
       let window = gWM.getMostRecentWindow("navigator:browser");
       if (window) {
+        let isOwnerInRoom = false;
+        // The participant that joined isn't necessarily included in room.participants (depending on
+        // when the broadcast happens) so concatenate.
+        for (let roomParticipant of room.participants.concat(participant)) {
+          if (roomParticipant.owner) {
+            isOwnerInRoom = true;
+            break;
+          }
+        }
+
+        let localizedString;
+        if (isOwnerInRoom) {
+          localizedString =
+            MozLoopServiceInternal.localizedStrings.get("rooms_room_joined_owner_connected_label2");
+        } else {
+          let l10nString = MozLoopServiceInternal.localizedStrings.get("rooms_room_joined_owner_not_connected_label");
+          let roomUrlDomain = new URL(room.decryptedContext.urls[0].location).hostname;
+          localizedString = PluralForm.get(1, l10nString).replace("#url", roomUrlDomain);
```

Comment:
Hmmm I've asked this morning on IRC how can I set the variable into the localized string because I'm not sure how can I access to the mozL10n in this file. Mark redirected me to that link about the PluralForm that's the reason why I'm using it.

AFAIK, `MozLoopServiceInternal.localizedStrings` it's just a map of the strings.